server/src/main/java/org/opensearch/index/translog/TranslogWriter.java

```diff
@@ -438,8 +438,43 @@ public TranslogReader closeIntoReader() throws IOException {
         synchronized (syncLock) {
             try (ReleasableLock toClose = writeLock.acquire()) {
                 synchronized (this) {
+                    Long translogWithoutFooterChecksum = null;
```

Comment:
nit: we don't need to explicitly initialise `Long` with `null` value

Reply:
I am more comfortable with overt initialisation of the variable prior and post the workflow. It find that it helps ensure that code is cleaner.

If you feel strongly about it or implicit initialisation leads to some performance benefits, we can change the same.